test/ConstructiveSolidGeometry/CSG_primitives.jl

```diff
@@ -205,4 +205,14 @@ no_translations = (rotation = one(SMatrix{3, 3, T, 9}), translation = zero(Carte
         @test !in(CartesianPoint{Float64}(1,1,3+tol),box_closed_trafo)
         @test in(CartesianPoint{Float64}(1,1,3-tol),box_open_trafo)
     end
+    @testset "TorusMantle" begin
+        # torusmantle1 = @inferred CSG.TorusMantle(r_torus=1f0)
```

Comment:
inferred does not work here due to D which can be :inwards or :outwards. I think we need to add two types instead of using symbols for inferred to work. Should we do that?

Reply:
@lmh91 ?

Reply:
Using types instead of symbols would not solve the issue. 

I think the best would be to remove `D` and add a field `normal_direction::T` (or some better name) to the struct.
Which either is `T(+1)` or `T(-1)`. 
This is related to #287. 

Just leave the test out for now.